assets/robotest/current/install.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: echoserver-install
+  namespace: default
+spec:
+  template:
+    metadata:
+      name: echoserver-install
+    spec:
+      restartPolicy: Never
```

Comment:
If the helm install fails, we have no reason to believe it would succeed if run again.

Reply:
While I was developing this, when `helm install` failed I found no reason to believe it would succeed when run a second, third, or fourth time.

`OnFailure` resulted in failures on an exponential backoff basis, resulting in longer timeouts before failure and a slower development cycle.  Perhaps if I knew more about the failure modes here I could make a more educated choice about whether to abort or retry.

Reply:
> when helm install failed I found no reason to believe it would succeed when run a second, third, or fourth time

Technically that's not entirely true, helm install can fail for example due to temporary etcd issues and succeed on the retry.

Reply:
Changed to `OnFailure` in 43c65ee.